### DISCUSSION TITLE
Geometry Points showing too few vertices

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#340] [Answers & Questions] [answered]
So I was following [this ](https://www.youtube.com/watch?v=mQN8VtVOltQ)tutorial on youtube and at around 3:15, a Rectangle OP is added where 100x100 size and 100x100 cols and rows are selected. I do the same. Then the GeometryPoints OP is supposed to show massive vertices. I only  have 12. [This ](https://cables.gl/edit/Erye_c)is the link of my patch. What am I doing wrong?

**Comment by pandrr** [comment, marked as answer]
looks fine to me, is it still happening ?

**Reply by monolithMktg** [reply]
Ok the problem is it is not updating for me in real time. I have to hard refresh the page every time to see the changes.  I'm on Firefox. What's missing?

Also, look at [this ](https://cables.gl/edit/NMmm_c)sketch. It is showing a weird behaviour. Followed one of the youtube tuts and it doesnt show the wireframe animation with all vertices. I remove and re attach some ops. It works. But then when I save and reload page, it is back to that simple square made of 2 triangles.

**Reply by pandrr** [reply]
you have to connect the yellow port so that the rectangle op gets updated

**Reply by monolithMktg** [reply]
@pandrr thank you very much. It must have been my mistake. You have an awesome piece of software there. I can't believe you have made it free for people like myself who are thoroughly enjoying the new found world of webgl. :)

I would love to play with it a little and then hopefully support Cables by creating youtube how-to vids :)
